### ISSUE TITLE
Caption regarding equilibrium conditions

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [pacific-hake/hake-assessment#1023]
https://github.com/pacific-hake/hake-assessment/blob/7af1d7bcca54bd09fc648e6ce99f7d259c456023/doc/main-tables/main-tables.rnw#L479--L487
```
Table 27. Summary of median and 95% credibility intervals of equilibrium conceptual reference points for
the base assessment model. Equilibrium reference points were computed using 2018–2022 averages for
mean weight-at-age and baseline selectivity (1966–1990; prior to time-varying deviations.)
```

I think "Equilibrium reference points were computed using 2018–2022 averages for
mean weight-at-age" is wrong but maybe I am confused?

**Comment by aaronmberger-nwfsc** [comment]
I think you are correct Kelli, equilibrium reference points would use "base" values for these (including selectivity).  The last five year language is for forecasts.  I guess the only time you would mention both like this would be when showing a metric calculated during a forecast year and comparing it to a reference point (i.e., our decision table scenario where we tune F=100%). I'll adjust the language in the table caption in the exec summary and main text.

**Comment by aaronmberger-nwfsc** [comment]
The executive summary table was correct, so just copied that language to this table.  Closed with 43696d6514f2b93fc83cc47ca28fc4891ab83caf
